app/scripts/lib/channels/mixins/postmessage_receiver.js

```diff
@@ -47,6 +47,8 @@ define([
 
       this.window = options.window || window;
 
+      this._origin = options.origin || '*';
```

Comment:
`*` is used by the fx-desktop channel. Maybe it should set it manually instead of setting it to `*` by default.